includes/class-micropub-render.php

```diff
@@ -8,12 +8,21 @@
  * Micropub Render Class
  */
 class Micropub_Render {
+
+	public static function get( $array, $key, $default = array() ) {
```

Comment:
this is already duplicated in two other places. we should probably make it a single, normal top-level function to avoid that.

Reply:
It will have to be prefixed then.

Reply:
ugh, right. so silly.

what do other plugins do about this? what does WordPress do? is their code just littered with `isset` checks? or do they structure it better somehow so they don't have to? or...?

Reply:
I don't think there is a better solution, but will look at structure

Reply:
I don't know how I ended up with conflicts, but here is my attempt at global functions.